app/values/scanned_resource_pdf/canvas_downloader.rb

```diff
@@ -17,22 +18,22 @@ def layout
       end
     end
 
-    private
+    def portrait?
+      canvas.width <= canvas.height
+    end
 
-      def portrait?
-        canvas.width <= canvas.height
-      end
+    private
 
       def canvas_url
-        "#{canvas.url}/full/!#{max_width},#{max_height}/0/#{quality}.jpg"
+        "#{canvas.url}/full/#{max_width},#{max_height}/0/#{quality}.jpg"
       end
 
       def max_width
-        792
+        [(612 * 1.5).round, canvas.width].min
```

Comment:
Unless you wanted to make the scale factor configurable. Regardless, would it make sense to make these numbers constants so that we know how they were determined?